subprojects/docs/src/docs/userguide/getting_started.adoc

```diff
@@ -0,0 +1,67 @@
+// Copyright 2018 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[getting_started]]
+= Getting Started
+
+Everyone has to start somewhere and if you're new to Gradle, then this is a good place to begin.
+
+== What is Gradle?
+
+Gradle is an open-source https://en.wikipedia.org/wiki/Build_automation[build automation] tool that is designed to be flexible enough to build almost any type of software. It:
+
+ * _Runs on the JVM_ — but it can build anything, including native libraries and executables
+ * _Runs fast_ — particularly with well-designed builds
+ * _Uses conventions and plugins_ — which make it simple to build common types of project
+ * _Is customizable and extensible_
+ * _Can be run from the command line or IDEs_ — like Android Studio, IntelliJ IDEA and Eclipse
+
+As with many flexible and customizable tools, Gradle can take a little time to understand. However, you can get a jump start by reading this short primer:
+
+ * <<five_things#five_things,5 things you need to know about Gradle>>
+
+== Short tutorials
+
+Actively using Gradle is a great way to learn about it, so try one of the introductory tutorials:
+
+ * {guidesUrl}/creating-new-gradle-builds/[Creating a basic Gradle build]
+ * {guidesUrl}/building-android-apps/[Building Android apps]
+ * {guidesUrl}/building-java-libraries/[Building Java libraries]
+ * {guidesUrl}/building-cpp-libraries/[Building C++ libraries]
```

Comment:
/me shrugs. I was just trying to think of the main ecosystems. If we add Kotlin, then the obvious question is why not Scala, Groovy, Swift, assembler (and so on) as well? I don't feel strongly about this, other than I don't want it to turn into a long list.

Reply:
@pledbrook Yes, please. Specifically Kotlin because the vast majority of Kotlin developers use Gradle. This is different from other languages list. We should consider removing C++ here based on that criteria, because that guide uses the deprecated software model.

Reply:
I'd argue that's a different criterion. Regardless, does the guide need updating for the new C++ plugins? Or are those plugins not ready for prime time yet?